### PR TITLE
Adding Ktoml library

### PR DESCRIPTION
**Pull request by akuleshov7**
# Library Title

Ktoml - MPP serialization library (decoder/encoder) for TOML format.

[Ktoml](https://github.com/akuleshov7/ktoml)

---

- [x] Confirmed that two spaces were added at the end of the description to break a new line.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,6 +145,17 @@
 ![badge][badge-watchos]
 ![badge][badge-windows]
 
+* [Ktoml](https://github.com/akuleshov7/ktoml) - MPP serialization library (decoder/encoder) for TOML format.
```
Comment (AAkira) [inline README.md:148]: Please add two spaces for a new line.

Comment (akuleshov7) [reply README.md]: Sorry, fixed after a long time 
I forgot about this PR :D

Comment (akuleshov7) [reply README.md]: @AAkira

**Comment by AAkira** [comment]
Thank you!

**Pull request merged**